packages/ui/src/components/Group/index.js

```diff
@@ -36,8 +36,10 @@ export const Strategies = {
 const defaultGetValue = (node) => node.props.value
 
 class GroupItem extends PureComponent {
+  static childrenProp = '$$groupItem'
+
   render() {
-    return <Fragment>{this.props.children}</Fragment>
+    return <Fragment>{this.props[GroupItem.childrenProp]}</Fragment>
```

Comment:
Minor stuff, but I think you can use the short syntax for `Fragment` here: `<>` and `</>`